#! /bin/sh
patch -p0 -f $* < $0
exit $?

The native Outlook 2003 S/MIME can't cope with AES.  Thus we change
the default back to 3DES.


--- sm/gpgsm.c  (revision 4923)
+++ sm/gpgsm.c  (working copy)
@@ -893,7 +893,7 @@
 
   /* Note: If you change this default cipher algorithm , please
      remember to update the Gpgconflist entry as well.  */
-  opt.def_cipher_algoid = "AES";  /*des-EDE3-CBC*/
+  opt.def_cipher_algoid = "3DES";
 
   opt.homedir = default_homedir ();
 
@@ -1607,7 +1607,7 @@
 #ifndef HAVE_W32_SYSTEM
         printf ("prefer-system-dirmngr:%lu:\n", GC_OPT_FLAG_NONE);
 #endif
-        printf ("cipher-algo:%lu:\"AES:\n", GC_OPT_FLAG_DEFAULT);
+        printf ("cipher-algo:%lu:\"3DES:\n", GC_OPT_FLAG_DEFAULT);
         printf ("p12-charset:%lu:\n", GC_OPT_FLAG_DEFAULT);
         printf ("default-key:%lu:\n", GC_OPT_FLAG_DEFAULT);
         printf ("encrypt-to:%lu:\n", GC_OPT_FLAG_DEFAULT);

